sanic/response.py

```diff
@@ -142,13 +140,45 @@ def cookies(self):
         return self._cookies
 
 
+class ContentRangeHandler:
+    """
+    This class is for parsing the request header
+    """
+    __slots__ = ('start', 'end', 'size', 'total', 'headers')
+
+    def __init__(self, request, stats):
+        self.size = self.start = 0
+        self.end = None
+        self.headers = dict()
+        self.total = stats.st_size
+        _range = request.headers.get('Range')
+        if _range is None:
+            return
```

Comment:
For these ones in particular would it be better to raise an exception here instead of returning `None` on an`__init__`? It seems like if anyone were to use this it'd be confusing if something went wrong in the middle.